src/Kernel.php

```diff
@@ -30,28 +30,28 @@ class Kernel extends BaseKernel
 {
     use MicroKernelTrait;
 
-    protected function configureContainer(ContainerConfigurator $container): void
+    protected function configureContainer(ContainerConfigurator $containerConfigurator): void
```

Comment:
What's the variable named in the parent class?  As of PHP 8.0 with named parameters it's a bad idea to rename method parameters, as it could break named-argument usage.  (Not relevant in this case, but a good habit to get into.)

Reply:
Hehe, currently the abstract definition is commented https://github.com/symfony/symfony/blob/5.4/src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php#L64